src/main/webapp/search_area.js

```diff
@@ -16,32 +16,52 @@ document.addEventListener("DOMContentLoaded", async function() {
   // This checks that requester User has valid credentials to edit/delete/view ALL organizations. (by checking userID)
   let isMaintainer = false;
   let forOrganizationsPage = false;
-
+  let forEventsPage = false;
   if (document.getElementById('search-area')) {
```

Comment:
This continues to be very odd for me. I believe the plan was to create a new SearchArea in js for each individual page, so you don't have the breakouts? This is otherwise leaking element ids?